reproductions/Directory.Build.props

```diff
@@ -4,6 +4,7 @@
     <PlatformTarget>$(Platform)</PlatformTarget>
     <RuntimeIdentifiers Condition="'$(TargetFramework)' == 'netcoreapp2.1'">win-x64;win-x86;linux-x64</RuntimeIdentifiers>
     <RuntimeIdentifier Condition="'$(TargetFramework)' != 'netcoreapp2.1'">win-$(Platform)</RuntimeIdentifier>
+    <TargetLatestRuntimePatch>false</TargetLatestRuntimePatch>
```

Comment:
This gives me the heebie-jeebies. Why are we adding it?

Reply:
Reading into some of the errors we're seeing, they're often caused by version mismatches for publish and restore. It was in DataDogThreadTest before and Zach added it to the TraceContext smoke test. It's mostly an experiment at the moment.

Reply:
I'm pretty sure we don't need `RuntimeIdentifier`/`RuntimeIdentifiers`
since netcore 2.1. Those might be causing those warnings.

Reply:
I'll try it out